testdata/p4_16_samples/bitvec_hdr.p4

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2018 MNK Consulting, LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <v1model.p4>
+
+struct alt_t {
+  bit<1> valid;
+  bit<7> port;
+};
+
+struct row_t {
+  alt_t alt0;
+  alt_t alt1;
+};
+
+header bitvec_hdr {
+  row_t row;
+}
+
+struct col_t {
+  bitvec_hdr bvh;
+}
+
+struct local_metadata_t {
+  row_t row0;
+  row_t row1;
+  col_t col;
+  bitvec_hdr bvh0;
+  bitvec_hdr bvh1;
+};
+
+struct parsed_packet_t {
+    bitvec_hdr bvh0;
+    bitvec_hdr bvh1;   
+};
+
+struct tst_t {
+  row_t row0;
+  row_t row1;
+  col_t col;  
+  bitvec_hdr bvh0;
+  bitvec_hdr bvh1;
+}
+
+parser parse(packet_in pk, out parsed_packet_t h,
+             inout local_metadata_t local_metadata,
+             inout standard_metadata_t standard_metadata) {
+  state start {
+    pk.extract(h.bvh0);
+    pk.extract(h.bvh1);
+    transition accept;
+  }
+}
+
+control ingress(inout parsed_packet_t h,
+                inout local_metadata_t local_metadata,
+	        inout standard_metadata_t standard_metadata) {
+
+    action do_act() {
+        h.bvh1.row.alt1.valid = 0;
+        local_metadata.col.bvh.row.alt0.valid = 0;	
+    }
+    
+    table tns {
+        key = {
+            h.bvh1.row.alt1.valid : exact;
+            local_metadata.col.bvh.row.alt0.valid : exact;
+        }
+	actions = {
+            do_act;
+        }
+    }
+    
+    apply {
+        tst_t s;
+        tns.apply();
+
+        h.bvh0.row.alt0.valid = 0;
+
+        s.row0.alt0 = local_metadata.row1.alt1;
+        s.row1.alt0.valid = 1;
+        s.row1.alt1.port = local_metadata.row0.alt1.port + 1;
+        s.col.bvh.row.alt0.valid = 0;
+
+        local_metadata.col.bvh.row.alt0.valid = 0;
```

Comment:
This code fragment does not seem to test anything related to the new code. It only uses nested structs.